morphuntion/src/morphuntion/Object.hpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2016-2024 Apple Inc. All rights reserved.
+ */
+#pragma once
```

Comment:
Hopefully this works everywhere we care about (we use guards instead). https://en.wikipedia.org/wiki/Pragma_once

Reply:
That page doesn't show any compilers that don't support it.  I guess the only issue would be if you want to use an old compiler.  From a maintenance point of view, this is much more reliable than using macros.